pebble_tool/commands/sdk/create.py

```diff
@@ -35,40 +37,57 @@ def __call__(self, args):
                 raise ToolError("A directory called '{}' already exists.".format(args.name))
             raise
 
+        project_template_path = os.path.join(self.get_sdk_path(), 'pebble', 'common', 'templates')
+        if not os.path.exists(project_template_path):
+            project_template_path = os.path.join(os.path.dirname(__file__), '..', '..', 'sdk', 'templates')
+
         # Create main .c file
-        with open(os.path.join(project_src, "{}.c".format(project_name)), "w") as f:
-            f.write(FILE_SIMPLE_MAIN if args.simple else FILE_DUMMY_MAIN)
+        if args.simple:
+            default_main = os.path.join(project_template_path, 'simple.c')
+        else:
+            default_main = os.path.join(project_template_path, 'main.c')
+        copy2(default_main, os.path.join(project_src, "{}.c".format(project_name)))
 
         # Add appinfo.json file
-        appinfo_dummy = DICT_DUMMY_APPINFO.copy()
-        appinfo_dummy['uuid'] = str(uuid.uuid4())
-        appinfo_dummy['project_name'] = project_name
+        with open(os.path.join(project_template_path, 'appinfo.json')) as f:
```

Comment:
This is a template string. In order to maintain the and formatting of the template, it'd probably be best to use that and just perform string substitution.